code/src/index.css

```diff
@@ -5,9 +5,169 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: rgb(24, 24, 20);
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+h1 {
+  color:white;
+  font-size: 1.5em;
+  margin: 30px 50px;
+}
+h2 {
+  color: #fff;
+  font-family: Helvetica, sans-serif;
+  font-size: 14px;
+}
+
+h2:hover {
+  text-decoration: underline;
+}
+
+p {
+  color: #a0a0a0;
+  font-family: Helvetica, sans-serif;
+  font-size: 14px;
+}
+p:hover {
+  text-decoration: underline;
+}
+a {
+  text-decoration: none;
+  color: #a0a0a0;
+  font-family: Helvetica, sans-serif;
+  font-size: 14px;
+}
+
+img:hover {
+  /* background: rgba(156, 166, 197, 0.7); */
+  background-color: black;
+}
+
+.overlay {
+  bottom: 0;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+  display:flex;
+  justify-content: center;
+}
+
+.overlay:hover {
+   /* Fallback color */
+   background-color: #1c1c1d;
+   background-color: rgba(8, 8, 8, 0.6);
+}
+.overlay:hover .Heart, .overlay:hover .Dots, .overlay:hover .Play {
+  fill: rgba(255, 255, 255, 0.742);
+  display: block;
+  margin: auto;
+  transition: 0.5s ease;
+}
+.overlay:hover .Play:hover {
+  height: 70px; 
+  width: 70px;
+
+}
+.overlay:hover .Play:hover, .overlay:hover .Heart:hover, .overlay:hover .Dots:hover {
+fill: white;
+
+}
+
+.img-container {
+  position:relative;
+}
+
+.container {
+  display: grid;
+  Grid-template-columns: 1fr;
+  grid-gap: 10px;
+  justify-items: center;
+/*   width: 320px;
+  margin:0 auto; */
+}
+.article {
+  grid-column: span 1;
+}
+
+/* These two under are made for trying the hover effects for the icons */
+.Heart, .Play, .Dots {
+  fill: white;
+  display: block;
+  margin: auto;
+  grid-column: span 1;
+  display: none;
+}
+.Play:hover {
+  height: 60px; 
+  width: 60px;
+  display: none;
+}
+
+
+/* Mobil -  */
+@media (min-width: 0px) and (max-width: 767px) { }
+
+
+/* Paddda -  */
+@media (min-width: 768px) and (max-width: 991px) {
+  h1 {
+    margin: 50px;
+    font-size: 2em;
+  }
+  
+  .container {
+    display: grid;
+    Grid-template-columns: 1fr 1fr;
+    grid-gap: 20px;
+    justify-items: center;
+  /*   width: 700px;
+    margin:0 auto; */
+  }
+  .article {
+    grid-column: span 1;
+  
+  }
+
+}
+
+
+/* Liten Dator -  */
+@media (min-width: 992px) {
+  h1 {
+  font-size: 3em;
+    
+}
+  
+  .container {
+    display: grid;
+    Grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 30px;
+    justify-items: center;
+   /*  width: 770px;
+    margin:0 auto; */
+  }
+  .article {
+    grid-column: span 1;
+  
+  }
+}
+/* Stor Dator -  */
```

Comment:
There is a solution to make it be 4 albums all the time (even on a small screen) if you are interested in seeing how that works you can ask us or just look in our code. That made us not have to do an extra css just for a smaller computer! Although we think this solutions is very nice aswell =)